home/dot_bashrc.d/create-pr.bash

```diff
@@ -1,10 +0,0 @@
-#######################################
```

Comment:
Re-create this but put under either `gh` or `git` command.

Maybe `gh create-pr`?

Reply:
#567